### PR TITLE
Updated privx-api to version 34.0.0

### PR DESCRIPTION
**Pull request by jjh-ssh**
## Status
Ready

## Description
Updated PrivX SDK for privx-api container to version 34.0.0

### CODE REVIEW

File: docker/privx-api/Dockerfile [hunk 1]
```diff
@@ -5,5 +5,5 @@ COPY requirements.txt .
 
 RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget git \
   && pip install --no-cache-dir -r requirements.txt \
+  && pip install git+https://github.com/SSHcom/privx-sdk-for-python.git@v34.0.0#egg=privx_api \
```
Comment (BEAdi) [inline docker/privx-api/Dockerfile:8]: Why is this needed? You do it in the Pipfile

**Review by BEAdi — COMMENTED** [review]
Hi,
Thank you for your contribution! 
I have a minor comment.
Is there also a related content PR? Can you tell me where this docker image in used?

File: docker/privx-api/Dockerfile (hunk 1 above)
Comment (jjh-ssh) [reply docker/privx-api/Dockerfile]: Correct, it's not needed here. Removed it.

**Comment by jjh-ssh** [comment]
This docker image is used by PrivX integration. The integration is not yet available in your repository, our customers have been using manual upload for installing the package. 
I will create a new PR to demisto/content repo as soon as this PR is merged and the new container image is available.

**Comment by jjh-ssh** [comment]
Doing some cleanup, closing the PR for now.
